atlasaction/action.go

```diff
@@ -725,6 +738,57 @@ func (a *Actions) SchemaPush(ctx context.Context) error {
 	return nil
 }
 
+// SchemaLint runs the GitHub Action for "ariga/atlas-action/schema/lint"
+func (a *Actions) SchemaLint(ctx context.Context) error {
+	tc, err := a.GetTriggerContext(ctx)
+	switch {
+	case err != nil:
+		return fmt.Errorf("unable to get the trigger context: %w", err)
+	}
+	params := &atlasexec.SchemaLintParams{
+		ConfigURL: a.GetInput("config"),
+		Vars:      a.GetVarsInput("vars"),
+		Env:       a.GetInput("env"),
+		URL:       a.GetArrayInput("url"),
+		Schema:    a.GetArrayInput("schema"),
+		DevURL:    a.GetInput("dev-url"),
+	}
+	report, err := a.Atlas.SchemaLint(ctx, params)
+	if err != nil {
+		a.SetOutput("error", err.Error())
+		return fmt.Errorf("`atlas schema lint` completed with errors:\n%s", err)
+	}
+	redactedURLs := make([]string, len(params.URL))
+	for i, u := range params.URL {
+		redacted, err := redactedURL(u)
+		if err != nil {
+			a.Errorf("failed to redact URL: %v", err)
+		}
+		redactedURLs[i] = redacted
+	}
```

Comment:
Return or skip the error element
```suggestion
	redactedURLs := make([]string, len(params.URL))
	for i, u := range params.URL {
		redacted, err := redactedURL(u)
		if err != nil {
			a.Errorf("failed to redact URL: %v", err)
		}
		redactedURLs[i] = redacted
	}
```